lib/lambda.js

```diff
@@ -2,6 +2,28 @@
 
 const shimmer = require('./instrumentation/shimmer')
 
+// used to detect cold starts
+let isFirstRun = false
```

Comment:
THis should default to `true`, shouldn't it?

Reply:
_nod_, it should.  I've changed the default value and added a test to catch similar problems in the future.